airbyte-webapp/babel.config.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  presets: [
+    ["@babel/preset-env", { targets: { node: "current" } }],
+    ["@babel/preset-react", { runtime: "automatic" }],
+    "@babel/preset-typescript",
+  ],
+  plugins: [
+    function () {
+      return {
+        visitor: {
+          MetaProperty(path) {
+            path.replaceWithSourceString("process");
```

Comment:
This will replace `import.meta` with `process`, e.g. `import.meta.env.VITE_FOO` > `process.env.VITE_FOO`